### PR TITLE
Fix dummy links position

### PR DESCRIPTION
**Pull request by BenjaminHug8**
Empty links in URDF are removed when the converter tool exports robot strings (otherwise we get `solid` without physics).

In case we have `Link1-joint1-DummyLink-joint2-Link2` in the URDF, the `joint1` and `joint2` has to be merged together in a third joint to get `Link1-joint3-Link2`.

This PR fix the resulting position when two joints are merged in one, add a test on this functionality and enable this functionality when exporting PROTO robots.

### CODE REVIEW

File: tests/expected/robotWithDummyLink.proto [hunk 1]
```diff
@@ -0,0 +1,103 @@
+#VRML_SIM R2022a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# This is a proto file for Webots for the robotWithDummyLink
+# Extracted from: /home/benjamin/urdf2webots/tests/sources/robotWithDummyLink.urdf
+
+PROTO robotWithDummyLink [
```
Comment (omichel) [inline tests/expected/robotWithDummyLink.proto:7]: PROTO are usually `UpperCamelCase` starting with a capital letter.

```suggestion
PROTO RobotWithDummyLink [
```
```suggestion
PROTO robotWithDummyLink [
```

File: tests/expected/robotWithDummyLink.proto [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  field  SFVec3f     translation     0 0 0
+  field  SFRotation  rotation        0 0 1 0
+  field  SFString    name            "robotWithDummyLink"  # Is `Robot.name`.
```
Comment (omichel) [inline tests/expected/robotWithDummyLink.proto:10]:
```suggestion
  field  SFString    name            "robot with dummy link"  # Is `Robot.name`.
```
```suggestion
  field  SFString    name            "robotWithDummyLink"  # Is `Robot.name`.
```

Comment (omichel) [inline tests/expected/robotWithDummyLink.proto:10]:
```suggestion
  field  SFString    name            "robot with dummy link"  # Is `Robot.name`.
```

Comment (omichel) [inline tests/expected/robotWithDummyLink.proto:10]:
```suggestion
  field  SFString    name            "robot with dummy link"  # Is `Robot.name`.
```

**Review by omichel — CHANGES_REQUESTED** [review]

**Comment by ad-daniel** [comment]
The name field in the PROTO cannot be changed as suggested. In the conversion it takes by default the name of the PROTO itself to fill this field, hence the two must coincide.

File: tests/sources/robotWithDummyLink.urdf [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<robot name="robotWithDummyLink">
```
Comment (omichel) [inline tests/sources/robotWithDummyLink.urdf:2]: For consistency, I would also use UpperCamelCase on this file.
```suggestion
<robot name="RobotWithDummyLink">
```

File: tests/test_export.py [hunk 4]
```diff
@@ -34,6 +34,12 @@
         'output': os.path.join(resultDirectory, 'KukaLbrIiwa14R820.proto'),
         'expected': [os.path.join(expectedDirectory, 'KukaLbrIiwa14R820.proto')],
         'arguments': '--box-collision --tool-slot=tool0 --rotation="1 0 0 -1.5708"'
+    },
+    {
+        'input': os.path.join(sourceDirectory, 'robotWithDummyLink.urdf'),
```
Comment (omichel) [inline tests/test_export.py:39]:
```suggestion
        'input': os.path.join(sourceDirectory, 'RobotWithDummyLink.urdf'),
```

**Comment by omichel** [comment]
The `robotWithDummyLink.urdf` file was not renamed

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**